libs/site-semantic-theme/src/semantic/thecoin/globals/site.overrides

```diff
@@ -72,6 +72,7 @@ body {
 /////////////////////////////////////////////////////////////
 // Links
 
+/*
 a{
     font-size: 12px;
```

Comment:
I don't actually have a spec for links.  We used to define all links using the front-page template, but since we converted that to a class (last PR) what a link should look like elsewhere is an open question